vendor/github.com/gravitational/trace/httplib.go

```diff
@@ -65,7 +65,9 @@ func ReadError(statusCode int, re []byte) error {
 	var e error
 	switch statusCode {
 	case http.StatusNotFound:
-		e = &NotFoundError{}
+		e = &NotFoundError{
+			Message: string(re),
+		}
 	case http.StatusBadRequest:
```

Comment:
this very same change has been submitted to `trace` repo as well. it will get overwritten when that PR is merged and trace is re-vendored.